pkg/gds/admin_test.go

```diff
@@ -86,6 +88,41 @@ func (s *gdsTestSuite) doRequest(handle gin.HandlerFunc, c *gin.Context, w *http
 	return res
 }
 
+// Test that the middleware returns the corect error when making unauthenticated
+// requests to protected endpoints.
+func (s *gdsTestSuite) TestAuthenticationMiddleware() {
+	s.svc.GetAdmin().SetHealth(true)
```

Comment:
Thought it might be useful to add a comment here.

```suggestion
	// We're not directly running the admin server so we need to manually set it as healthy       
	s.svc.GetAdmin().SetHealth(true)
```